goconvey.go

```diff
@@ -90,21 +91,19 @@ func folders() {
 			}
 		} else {
 			if err != nil && stat == nil {
-				mkdir(folder)
+				if os.IsNotExist(err) {
+					mkdir(folder)
+				} else {
+					log.Fatal(err)
+				}
 			}
 		}
 	}
 
-	tmpStaticPath := filepath.Join(tmpDest, "web", "client")
-	cleanupAndMkdir(tmpStaticPath)
 	tmpReportsPath := filepath.Join(tmpDest, "reports")
 	cleanupAndMkdir(tmpReportsPath)
 
-	if err := copy.Copy(localStatic, tmpStaticPath); err != nil {
-		log.Fatal(err)
-	}
-
-	static = tmpStaticPath
+	static = localStatic
```

Comment:
@riannucci localStatic works just fine...
Also added  `os.IsNotExist(err)` and `os.IsExist(err)` to isolate the respective exceptions that can be safely ignored, or Fatal'ed explicitly.